fem/lor/lor_h1.cpp

```diff
@@ -531,29 +544,8 @@ BatchedLOR_H1::BatchedLOR_H1(BilinearForm &a,
                              Array<int> &sparse_mapping_)
    : BatchedLORKernel(fes_ho_, X_vert_, sparse_ij_, sparse_mapping_)
 {
-   MassIntegrator *mass = GetIntegrator<MassIntegrator>(a);
-   DiffusionIntegrator *diffusion = GetIntegrator<DiffusionIntegrator>(a);
-
-   if (mass != nullptr)
-   {
-      auto *coeff = dynamic_cast<const ConstantCoefficient*>(mass->GetCoefficient());
-      mass_coeff = coeff ? coeff->constant : 1.0;
-   }
-   else
-   {
-      mass_coeff = 0.0;
-   }
-
-   if (diffusion != nullptr)
-   {
-      auto *coeff = dynamic_cast<const ConstantCoefficient*>
-                    (diffusion->GetCoefficient());
-      diffusion_coeff = coeff ? coeff->constant : 1.0;
-   }
-   else
-   {
-      diffusion_coeff = 0.0;
-   }
+   ProjectLORCoefficient<MassIntegrator>(a, c1);
```

Comment:
Just to double check I understand, we don't have to sample coefficient data for the LOR spaces? This PR will do it for us?

Reply:
For LOR preconditioning, the coefficients are evaluated at the Gauss-Lobatto points of the high-order space. Does that make sense?